index.html

```diff
@@ -389,6 +389,12 @@ <h2 class="title-bar-item">JSON Path Finder</h2>
   }
 });
 
+document.querySelectorAll(".big-box").forEach((el) => { el.style.height= (window.innerHeight - 90) + "px" });
+
+addEventListener('resize', (event) => {
```

Comment:
nit: just to denote that this param is unused
```suggestion
addEventListener('resize', (_event) => {
```

Reply:
I was unaware of this convention, but recall seeing it before now; thanks for the 💡